src/main/java/io/kuzzle/sdk/core/Collection.java

```diff
@@ -1547,6 +1784,57 @@ public void onError(JSONObject error) {
   }
 
   /**
+   * Validates the provided specifications
+   *
+   * @param specifications JSONObject Specifications content
+   * @param listener Response callback
+   */
+  public void validateSpecifications(@NonNull JSONObject specifications, @NonNull ResponseListener<Boolean> listener) throws JSONException {
+    this.validateSpecifications(specifications, new Options(), listener);
+  }
+
+  /**
+   * Validates the provided specifications
+   *
+   * @param specifications JSONObject Specifications content
+   * @param options Options Optional parameters
+   * @param listener Response callback
+   */
+  public void validateSpecifications(@NonNull JSONObject specifications, Options options, @NonNull final ResponseListener<Boolean> listener) throws JSONException {
```

Comment:
You should check if listener is null and throw an IllegalArgumentException if so